src/components/Pagination/Pagination.tsx

```diff
@@ -31,9 +31,15 @@ export const Pagination = ({
   onChange: onChangeProp,
   onPageSizeChange: onPageSizeChangeProp,
   pageSize = -1,
+  fillWidth = true,
+  gap = "md",
+  justifyContent,
   ...props
 }: PaginationProps): ReactElement => {
   const hasRowCount = ["number", "string"].includes(typeof rowCount);
+  const formatNumber = (value: number) => {
+    return new Intl.NumberFormat("en").format(value);
```

Comment:
not sure we should render it the American way for everyone in the world. We should probably localize this for the country of the user.